src/broker/Manager.h

```diff
@@ -143,6 +143,13 @@ class Manager : public iosource::IOSource
 	 */
 	void Peer(const std::string& addr, uint16_t port, double retry = 10.0);
 
+	/**
+	 * Initiate a peering with a remote endpoint but tries only once.
```

Comment:
Is the intention for this method to only be used by tests? We probably should mention that if so.

Reply:
I am going to assume that the answer is "yes", and merge it with a comment added. If this is incorrect, we can remove the comment later.

(Getting annoyed about the test failures on my machine)